### ISSUE TITLE
Xplorer .AppImage doesn't run on Fedora Linux

### ISSUE THREAD

**Issue by akza07** [kimlimjustin/xplorer#255]
### Description

Using Fedora Linux 36 Work Station, when I try to launch the AppImage it throws these messages.

```
Gtk-Message: 21:26:42.869: Failed to load module "canberra-gtk-module"
Gtk-Message: 21:26:42.870: Failed to load module "pk-gtk-module"
Gtk-Message: 21:26:42.871: Failed to load module "canberra-gtk-module"
Gtk-Message: 21:26:42.872: Failed to load module "pk-gtk-module"

** (xplorer:31336): ERROR **: 21:26:42.996: Unable to spawn a new child process: Failed to execute child process “/usr/lib/x86_64-linux-gnu/webkit2gtk-4.0/WebKitNetworkProcess” (No such file or directory)
[1]    31336 trace trap (core dumped)  ./xplorer_0.3.1_amd64.AppImage
➜  Downloads GDB_BACKEND=x11 ./xplorer_0.3.1_amd64.AppImage
Gtk-Message: 21:29:18.698: Failed to load module "canberra-gtk-module"
Gtk-Message: 21:29:18.699: Failed to load module "pk-gtk-module"
Gtk-Message: 21:29:18.700: Failed to load module "canberra-gtk-module"
Gtk-Message: 21:29:18.701: Failed to load module "pk-gtk-module"

** (xplorer:31668): ERROR **: 21:29:18.818: Unable to spawn a new child process: Failed to execute child process “/usr/lib/x86_64-linux-gnu/webkit2gtk-4.0/WebKitNetworkProcess” (No such file or directory)
[1]    31668 trace trap (core dumped)  ./xplorer_0.3.1_amd64.AppImage
```
It was the same in Fedora 35 Workstation.

### Steps To Reproduce

just make it executable and run it. No UI or Error and fails silently.
Use terminal to run, it shows the given messages and ends.

### Expected behavior

App to launch and show a GUI.

### Xplorer Version

v0.3.1

### Operating System Version

Fedora 36 Workstation

### Additional Information

![image](https://user-images.githubusercontent.com/30655539/168482424-43863b0c-44c4-457b-9a81-0ace53373b45.png)

**Comment by catendy** [comment]
Can confirm this happening to me too under Arch Linux. I have installed all the webkit2gtk versions in the official repos but the "/usr/lib/x86_64-linux-gnu" folder doesn't exist. I do not get the gtk canberra messages tho.

**Comment by akza07** [comment]
It seems `libwebkit2gtk-4.0-37` package needed as Tauri dependency isn't installed in Most distribution.

In fedora's case, running
`sudo dnf install libwebkit2gtk-4.0-37` gives
`No match for argument: libwebkit2gtk-4.0-37
Error: Unable to find a match: libwebkit2gtk-4.0-37`

It's not in the official repos.

I think the AppImage will have to include libwebkit2gtk itself which I'm unsure Xplorer devs should handle or Tauri's side.

**Comment by kimlimjustin** [comment]
Really sorry that I won't able to check which problems is it but one question, does installing `libwebkit2gtk` fix it? If it does, I think I'll just write it on the docs (or maybe any PR will be welcomed)

**Comment by catendy** [comment]
I did have installed the dependency on my system, the correct version. However it expected it to be under "/usr/lib/x86_64-linux-gnu/" and I had it under "/usr/lib/". I did a quick test with a symlink and it works just fine, the application now works, but that's not a solution.

Maybe the distro doesn't have the dependency or maybe it has but it's just in a different location, but that dependency, being a critical dependency which has caused problems, should be in the AppImage itself, I think.

**Comment by akza07** [comment]
I don't have that package available in my Distribution's repository. I asked them in the forum. But is there way you can ship the `libwebkit2gtk-4.0-37` in the Appimage? Since app images are supposed to come with most of the dependencies in it. Or is the AppImage built automatically by Tauri's scripts?

**Comment by akza07** [comment]
Yes. That [Pull request ](https://github.com/tauri-apps/tauri/pull/2940)to Tauri also mentions my case. So it's a Tauri issue.

**Comment by Mart-Bogdan** [comment]
App image is broken. Manjaro Linux.

For some reason you have in readme badge of [Arch] which points not to AUR but to releases page on GH. And there are no tgz, only stupid AppImage, and deb package.


When I run AppImage I'm gerring following error:

```
Unable to spawn a new child process: Не вдалося виконати дочірній процес «/usr/lib/x86_64-linux-gnu/webkit2gtk-4.0/WebKitNetworkProcess» (No such file or directory)

```

Wasn't it the point of app image to bundle everything? 

And wasn't this the point of Tauri -- to use system PROVIDED webkit, to not increase package size.


And then I unpacked deb archive and run it, and it works!!!

Previously I have installed webkitgtk package from repo.

**Comment by ZHassanQ** [comment]
I tried all the versions and 1.0 is the only version worked. 

I use Fedora 37 and installed `sudo dnf install webkit2gtk3` and still the latest versions don't work

**Comment by Mart-Bogdan** [comment]
@ZHassanQ I think you need GTK4, not GTK3

**Comment by Mart-Bogdan** [comment]
But app image is broken. Just try unpacking deb package.

That's a shame, that `Xplorer.app.tar.gz` is Mac version. Would be nice to have linux version packaged as simple archive, and let users to take care of their dependencies.

**Comment by ZHassanQ** [comment]
@Mart-Bogdan 'sudo dnf install gtk4' says is already installed.
